packages/algorand-js/test/integration/escrow-account.ts

```diff
@@ -0,0 +1,89 @@
+/* eslint sonarjs/no-duplicate-string: 0 */
+import { assert } from "chai";
+
+import { ERRORS } from "../../src/errors/errors-list";
+import { Runtime } from "../../src/index";
+import { StoreAccountImpl } from "../../src/runtime/account";
+import { getAcc } from "../helpers/account";
+import { expectTealErrorAsync } from "../helpers/errors";
+import { johnAccount } from "../mocks/account";
+
+describe("Algorand Stateless Smart Contracts", function () {
+  const escrow = new StoreAccountImpl(1000000000); // 1000 ALGO
```

Comment:
Use variables for numbers:
```
const initialAlgoHolding = 100e6
```
and use it in tests below.  In checks you can use: `assert.equal(..., initialAlgoHolding - 100)`

Reply:
ok, thanks